### PR TITLE
Feature: create Internship Page

### PR DESCRIPTION
**Pull request by OstroumovAlex**
https://trello.com/c/HVFH5gDB/97-fe-create-internship-page
image

### CODE REVIEW

**Review by svtslv17 — APPROVED** [review]

File: src/constants/tableInternshipPageFields.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export const columnDefsMentors = 
+[
+  {"headerNames": "Full Name", "field": "fullName"},
+  {"headerNames": "Position", "field": "position"},
```
Comment (itsnotpolza) [inline src/constants/tableInternshipPageFields.js:4]: use labels fir header names like in the candidates page

File: src/pages/internshipPage/InternshipPage.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      <Box className="internshipContentWrapper">
+        <Box className="tableWrapper" >
+          <Card className="InternshipDataMentors">
+            <Typography variant="h4" >{getFieldLabel('internship.page.mentors')}</Typography>
+            <ProfileTable  rowData={rowDataMentors} columnDefs={columnDefsMentors} />
+          </Card>
+        </Box> 
+      </Box>  
+      <Box className="internshipContentWrapper">
+        <Box className="tableWrapper" >
```
Comment (itsnotpolza) [inline src/pages/internshipPage/InternshipPage.jsx:84]: This could be one box

File: src/pages/internshipPage/internshipPage.sass [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  height: 100%
+  padding: 1%
+  padding-right: 0
+  box-sizing: border-box
+
+.tableWrapper      
+  height: 100%
+  box-sizing: border-box
+
+.InternshipDataText
```
Comment (itsnotpolza) [inline src/pages/internshipPage/internshipPage.sass:20]: Classnames should start with a lowercase letter

File: src/components/feedbacks/ProfileTable.jsx [hunk 4]
```diff
@@ -12,11 +12,18 @@ export default class ProfileTable extends React.Component {
   }
 
   render() {
+    const { checkboxSelection } = this.props;
     const { columnDefs } = this.props;
     const { rowData } = this.props;
+    const { pagination } = this.props;
+    const { paginationPageSize } = this.props;
```
Comment (itsnotpolza) [inline src/components/feedbacks/ProfileTable.jsx:19]:
```suggestion
    const { checkboxSelection, paginationPageSize, columnDefs, rowData, pagination } = this.props;
```

File: src/pages/internshipPage/InternshipPage.jsx [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+  rowDataEmployees,
+  internshipData,
+} from '../../mocks/internshipEmployees.json';
+import { columnDefsEmployees } from '../../constants';
+import ProfileTable from '../../components/feedbacks/ProfileTable';
+import { useMediaDown } from '../../components/utils';
+
+const InternshipPage = () => {
+  const { internshipJavascript } = assets;
+  const mobile = useMediaDown('800');
```
Comment (nsukhotskaya) [inline src/pages/internshipPage/InternshipPage.jsx:25]: Instead of '800' use one of them: xs, sm, md, lg

**Review by nsukhotskaya — APPROVED** [review]

**Review by beliavskaja — APPROVED** [review]

**Review by svtslv17 — APPROVED** [review]

**Pull request merged**